libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -386,6 +386,11 @@
         "comment": "",
         "limit": 0
     },
+    "gs.header.inviteButtonText": {
```

Comment:
Please rename `"gs.header.inviteButtonText"` → `"gs.header.invite"`, as the item is not a button, is it?